### PR TITLE
Add Dockerfile.

### PR DESCRIPTION
**Pull request by uphy**
Add Dockerfile/docker-compose for trying it out or developing redis-nativemsgpack quickly.

### CODE REVIEW

File: docker/docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+
+services:
+  redis:
+    build:
+      context: ..
+      dockerfile: docker/Dockerfile
+    ports:
+      - 6379:6379
+    volumes:
+      - ./docker/data:/data
```
Comment (ocadaruma) [inline docker/docker-compose.yml]: By this, `docker/docker/data` was created.

**Review by ocadaruma — COMMENTED** [review]
Thank you for submitting PR !
Please take a look my comment.

File: docker/docker-compose.yml (hunk 1 above)
Comment (uphy) [reply docker/docker-compose.yml]: Thanks for the comment.
I fixed with this commit: d7b5b1cf93c41fd02308d2f3a531dfb06513e326

**Comment by ocadaruma** [comment]
LGTM, Thank you !

**Pull request merged**